clients/vscode/src/chat/ChatViewProvider.ts

```diff
@@ -313,10 +323,25 @@ export class ChatViewProvider implements WebviewViewProvider {
     if (serverInfo.config.token) {
       this.client?.cleanError();
       // Duplicate init won't break or reload the current chat page
+      const focusKey = contributes.keybindings.find((cmd) => cmd.command === "tabby.chatView.focus");
```

Comment:
The `contributes` is import from `package.json` at build time, so it is always `ctrl/cmd+l`. So if we are aiming to support the user-customized keybinding, it is required to read the vscode user keybinding config file.

Reply:
I'll fix it rn